grudge/models/em.py

```diff
@@ -332,7 +332,7 @@ def get_eh_subset(self):
         """
         return 6*(True,)
 
-    def max_eigenvalue_expr(self):
+    def max_characteristic_vel_expr(self):
```

Comment:
Why have this (instead of just `max_characteristic_velocity`)? I think this may have made sense in the symbolic world, but not so much any more.

Reply:
Done! https://github.com/inducer/grudge/pull/108/commits/92607dfbc85b2e1274ec42c781976c989bf1ec0b